lib/new-admin/admin-server.js

```diff
@@ -66,34 +102,7 @@ app.use(cors({ credentials: true, origin: devMode && 'https://localhost:3001' })
 app.use('/id-card-photo', serveStatic(idPhotoCardBasedir, { index: false }))
 app.use('/front-camera-photo', serveStatic(frontCameraBasedir, { index: false }))
 
-app.get('/api/register', (req, res, next) => {
-  const otp = req.query.otp
-  const ua = req.headers['user-agent']
-  const ip = req.ip
-
-  if (!otp) return next()
-
-  return login.register(otp, ua, ip)
-    .then(r => {
-      if (r.expired) return res.status(401).send('OTP expired, generate new registration link')
-
-      // Maybe user is using old registration key, attempt to authenticate
-      if (!r.success) return next()
-
-      const cookieOpts = {
-        httpOnly: true,
-        secure: true,
-        domain: hostname,
-        sameSite: true,
-        expires: NEVER
-      }
-
-      const token = r.token
-      req.token = token
-      res.cookie('token', token, cookieOpts)
-      res.sendStatus(200)
-    })
-})
+require('./routes/auth')(app)
```

Comment:
Take a look at how Cesar is handling requiring routes and middlewares in https://github.com/lamassu/lamassu-server/pull/563/files, seems to be the standard